content/working_with/console/widgets/deploymentsView.md

```diff
@@ -9,8 +9,8 @@ Presents information about deployments in a single view. Contains:
 
 - a table showing the deployments in the system (the left side)
 
-    The deployments can be further filtered.
-    See [Filtering deployments]({{< relref "#filtering-deployments" >}}).
```

Comment:
When I was doing the review, I hide whitespaces changes, but now I realized that probably such changes introduces indent change, so please verify if they are expected.

Use the link to the documentation site deployed from your branch I shared with the other comment or install `hugo` framework locally (installation and setup iss described here: https://github.com/cloudify-cosmo/docs.getcloudify.org#installing-the-cloudify-documentation-center).

Reply:
I've restored the previous whitespaces, so that the issue wouldn't exists :)